deeplake/api/dataset.py

```diff
@@ -232,8 +283,16 @@ def exists(
 
         Returns:
             A boolean confirming whether the dataset exists or not at the given path.
+
+        Raises:
+            ValueError: If version is specified in the path
         """
-        path = convert_pathlib_to_string_if_needed(path)
```

Comment:
is this function no longer required? we should get ride of it if so

Reply:
it is used by other functions, however it doesn't need to be used together with `process_dataset_path`.